addons/website_slides/data/gamification_data.xml

```diff
@@ -0,0 +1,207 @@
+            <field name="visibility_mode">personal</field>
+            <field name="report_message_frequency">never</field>
+            <field name="reward_id" eval="ref('badge_data_certification')"/>
+            <field name="reward_realtime">True</field>
+            <field name="user_domain">[('karma', '>', 0)]</field>
+            <field name="state">inprogress</field>
+        </record>
+        <record id="badge_data_certification_challenge_line_0" model="gamification.challenge.line">
+            <field name="definition_id" eval="ref('badge_data_certification_goal')"/>
+            <field name="challenge_id" eval="ref('badge_data_certification_challenge')"/>
+            <field name="target_goal">1</field>
+        </record>
+
+        <!-- Community hero: read 2000 XP -->
+        <record id="badge_data_karma" model="gamification.badge">
+            <field name="name">Community hero</field>
+            <field name="description">Reach 2000 XP</field>
+            <field name="image" type="base64" file="website_slides/static/src/img/standard_badge_gold.svg"/>
+            <field name="website_published" eval="True"/>
+            <field name="level">gold</field>
+            <field name="rule_auth">nobody</field>
+        </record>
+        <record id="badge_data_karma_goal" model="gamification.goal.definition">
+            <field name="name">Community hero</field>
+            <field name="description">Reach 2000 XP</field>
+            <field name="computation_mode">count</field>
+            <field name="display_mode">boolean</field>
+            <field name="model_id" eval="ref('base.model_res_users')"/>
+            <field name="condition">higher</field>
+            <field name="domain">[
+                ('karma', '>=', 2000)
+            ]</field>
+            <field name="batch_mode">True</field>
+            <field name="batch_distinctive_field" eval="ref('base.field_res_users__id')"/>
+            <field name="batch_user_expression">user.id</field>
+        </record>
+        <record id="badge_data_karma_challenge" model="gamification.challenge">
+            <field name="name">Read 2000 XP</field>
+            <field name="category">slides</field>
+            <field name="period">once</field>
+            <field name="visibility_mode">personal</field>
+            <field name="report_message_frequency">never</field>
+            <field name="reward_id" eval="ref('badge_data_karma')"/>
+            <field name="reward_realtime">True</field>
+            <field name="user_domain">[('karma', '>', 0)]</field>
+            <field name="state">inprogress</field>
+        </record>
+        <record id="badge_data_karma_challenge_line_0" model="gamification.challenge.line">
+            <field name="definition_id" eval="ref('badge_data_karma_goal')"/>
+            <field name="challenge_id" eval="ref('badge_data_karma_challenge')"/>
+            <field name="target_goal">1</field>
+        </record>
+</data></odoo>
```

Comment:
lint last line :) also applies in some other files

Reply:
Done